src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Button = (props) => {
+  return (
+    <button onClick = {props.onClick} className = {props.className}>
+      {props.value}
+    </button>
+  )
+}
```

Comment:
implement `propTypes` to that `value` props is STRING
- https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png